include/inviwo/core/processors/progressbar.h

```diff
@@ -66,8 +68,8 @@ class IVW_CORE_API ProgressBarObserver : public Observer {
  */
 class IVW_CORE_API ProgressBarObservable : public Observable<ProgressBarObserver> {
 protected:
-    void notifyProgressChanged();
-    void notifyVisibilityChanged();
+    void notifyProgressChanged(float progress);
+    void notifyVisibilityChanged(bool visible);
```

Comment:
When you are changing the interface you need to make sure to cover all places where it is used!
E.g. in `ProcessorProgressGraphicsItem`

https://github.com/inviwo/inviwo/blob/6cadbb3e15476da324c0faa83847c8c8ed2ebcb4/include/inviwo/qt/editor/processorprogressgraphicsitem.h#L57-L58

Reply:
Nasty, it compiled for me. I guess it is because they lack override